beta/src/components/MDX/ExpandableExample.tsx

```diff
@@ -8,23 +8,21 @@ import {IconChevron} from '../Icon/IconChevron';
 import {IconDeepDive} from '../Icon/IconDeepDive';
 import {IconCodeBlock} from '../Icon/IconCodeBlock';
 import {Button} from '../Button';
+import {H4} from './Heading';
 
 interface ExpandableExampleProps {
   children: React.ReactNode;
-  title: string;
   excerpt?: string;
   type: 'DeepDive' | 'Example';
 }
 
-function ExpandableExample({
-  children,
-  title,
-  excerpt,
-  type,
-}: ExpandableExampleProps) {
+function ExpandableExample({children, excerpt, type}: ExpandableExampleProps) {
   const [isExpanded, setIsExpanded] = React.useState(false);
   const isDeepDive = type === 'DeepDive';
   const isExample = type === 'Example';
+  const hasTitle =
```

Comment:
I would like to raise an error if the title is not found. Also let's force it to always be `h4`. Basically we should be strict about the format so that it's impossible to write a DeepDive without a proper header.

Another way you can do this, btw, is to preprocess it in `prepareMDX` and generate a regular `title` prop there. But I guess for now this is fine.